Author: Simon Frankenberger <simon-alpine@fraho.eu>
Subject: Fix compilation of jtreg / gtest objects with -Werror=format-security CFLAGS
Upstream: Yes / closed unresolved (https://bugs.openjdk.org/browse/JDK-8218814)

Remove the -Wno-format CFLAG for test objects

--- old/make/common/TestFilesCompilation.gmk	2023-07-11 17:28:44.345270760 +0200
+++ new/make/common/TestFilesCompilation.gmk	2023-07-11 17:29:57.941716556 +0200
@@ -99,7 +99,7 @@
         CFLAGS := $$($1_BASE_CFLAGS) $$($1_CFLAGS) $$($1_CFLAGS_$$(name)), \
         CXXFLAGS := $$($1_BASE_CXXFLAGS) $$($1_CFLAGS) $$($1_CFLAGS_$$(name)), \
         LDFLAGS := $$($1_LDFLAGS) $$($1_LDFLAGS_$$(name)), \
-        DISABLED_WARNINGS_gcc := format undef unused-function unused-value, \
+        DISABLED_WARNINGS_gcc := undef unused-function unused-value, \
         DISABLED_WARNINGS_clang := undef format-nonliteral \
             missing-field-initializers sometimes-uninitialized, \
         LIBS := $$($1_LIBS_$$(name)), \
